style.css

```diff
@@ -508,4 +508,63 @@ main > div img {
  div.evidon-notice-link {
    display:none;
  }
- 
\ No newline at end of file
+ 
+  /*
+  * Styles for Dropdown Region Selector
+  */
+.region-dropdown {
+  border: 1px solid #d3d3d3;
+  background-color: #ffffff;
+  border-radius: 4px;
+  display: inline-block;
+  font-size: inherit;
+  padding: 3px 15px 5px 8px;
+  margin: 0;
+  visibility: hidden;
+  position: absolute;
+  z-index: 0;
+}
+
+.region-dropdown.visible {
+  visibility: visible;
+  z-index: 2;
+}
+
+.region-dropdown ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a.region-dropdown-picker {
+  color: #505050;
+  font-size: .9rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+}
+
+a.region-dropdown-picker:after {
+  display: block;
+  margin-left: 15px;
+  color:rgba(20, 115, 230, 0);
+  transform-origin: center center;
+  transform: scaleX(2);
+  content: "\2228";
+  font-size: .9rem;
+}
+
+a.region-dropdown-picker.selected:after {
```

Comment:
This looks identical to `a.region-dropdown-picker:after` above... do we really need both?

Reply:
CSS was updated, selected status was simplified